google/cloud/opentelemetry/trace_exporter.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 Google LLC
```

Comment:
This file seems unrelated to the docs, but meh.

Reply:
In case you missed the PR description:

> Without the trace_exporter.h file, Doxygen would not have any input. (It skips internal/ directories). And if Doxygen does not have input, it throws an error.

I'd rather set up the doxygen up front and add a temporarily dumb file than have to remember to add doxygen later. :shrug: